test/functional/api/cas/statistics.py

```diff
@@ -0,0 +1,643 @@
+                stats["cleaning policy"],
+                stats["promotion policy"],
+                stats["cache line size"],
+                stats["metadata memory footprint"],
+                stats["dirty for"],
+                stats["metadata mode"],
+                stats["status"],
+            )
+        except KeyError:
+            pass
+        try:
+            self.usage_stats = UsageStats(
+                stats["occupancy"], stats["free"], stats["clean"], stats["dirty"]
+            )
+        except KeyError:
+            pass
+        try:
+            self.inactive_usage_stats = InactiveUsageStats(
+                stats["inactive occupancy"] if "inactive occupancy" in stats else None,
+                stats["inactive clean"] if "inactive clean" in stats else None,
+                stats["inactive dirty"] if "inactive dirty" in stats else None,
+            )
+        except KeyError:
+            pass
+        try:
+            self.request_stats = RequestStats(
+                stats["read hits"],
+                stats["read partial misses"],
+                stats["read full misses"],
+                stats["read total"],
+                stats["write hits"],
+                stats["write partial misses"],
+                stats["write full misses"],
+                stats["write total"],
+                stats["pass-through reads"],
+                stats["pass-through writes"],
+                stats["serviced requests"],
+                stats["total requests"],
+            )
+        except KeyError:
+            pass
+        try:
+            self.block_stats = BlockStats(
+                stats["reads from core(s)"],
+                stats["writes to core(s)"],
+                stats["total to/from core(s)"],
+                stats["reads from cache"],
+                stats["writes to cache"],
+                stats["total to/from cache"],
+                stats["reads from exported object(s)"],
+                stats["writes to exported object(s)"],
+                stats["total to/from exported object(s)"],
+            )
+        except KeyError:
+            pass
+        try:
+            self.error_stats = ErrorStats(
+                stats["cache read errors"],
+                stats["cache write errors"],
+                stats["cache total errors"],
+                stats["core read errors"],
+                stats["core write errors"],
+                stats["core total errors"],
+                stats["total errors"],
+            )
+        except KeyError:
+            pass
+
+    def __str__(self):
+        status = ""
+        for stats_item in self.stats_list:
+            if stats_item == "inactive_usage_stats" and (
+                self.inactive_usage_stats.inactive_occupancy is None
+                and self.inactive_usage_stats.inactive_clean is None
+                and self.inactive_usage_stats.inactive_dirty is None
+            ):
+                stats_item = ""
```

Comment:
Is this piece of code really necessary? Why do we need special case for inactive stats?